api/include/can_protocols/brake_can_protocol.h

```diff
@@ -13,6 +13,13 @@
 #include "magic.h"
 
 
+/*
+ * @brief CAN ID representing the range of brake messages.
+ *
+ */
+#define OSCC_BRAKE_CAN_ID_INDEX (0x50)
```

Comment:
Small nit: this is a mask for CAN IDs, and not a CAN id itself right?

Reply:
It's what I'm calling the CAN ID index, meaning it's where the range of CAN IDs for that module start.

Reply:
Can we just call it that then? `OSCC_BRAKE_CAN_ID_RANGE_START` or something?